### DISCUSSION TITLE
Is it planned to support differents gauges and track equipments ?

### DISCUSSION THREAD

**Discussion by CyberFox001** [Libre-TrainSim/Libre-TrainSim#447] [Ideas]
Hello,

First, thank you for your amaizing work. :)

I know it's maybe too early to ask it, but do you plan to support differents gauges and track equipments ?

For example, in the map editor:
  * A "gauge" setting for the tracks
  * Again in the track settings, an option to select the catenary
  * When adding a signaling light, having differents track equipments libraries (German, French, Swiss, etc)
  * Ability to choose the track exploitation rules, between rules of differents countries (German, French, Swiss, etc)

**Comment by HaSa1002** [comment]
'ello,
sorry for forgetting your question.
So the first two are likely to be answered once we rework the tracks themselves. It' s visually possible but since there's little simulation anyway its not taken into account for game play. The same goes for the catenaries. There isn't even a check for electrification right now.

Regarding signalling: It'll require further UX improvements that we want to tackle at some point but usage of other signals is already possible. Once unified with the build menu you can also create country resource packs.

I am not so sure what you mean with track exploitation though. I figure it's regarding the usage of trains on thr tracks and rules? If so, I am not sure that's the kinda realism we're aiming for. I, for my part, am always gonna push for a good experience rather than a realistic one

**Reply by CyberFox001** [reply]
Thank you very much for your reply. Hope you're doing good. :heart: 

Yes, for the 4th point I was talking about how the different countries manages theirs tracks. The side they travel on when there is 2 tracks, if they need to confirm some signals, etc.
